extensions/kube-object-event-status/package/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Is this file added by accident?

Reply:
Yes

Reply:
@Nokel81 just noticed that this file was not removed ...